src/components/UserInfo/UserInfo.scss

```diff
@@ -1 +1,3 @@
-//
```

Comment:
Consider adding a general comment at the beginning of the file to briefly describe the purpose of the UserInfo component.